framework/messages/hi/yii.php

```diff
@@ -0,0 +1,138 @@
+    'Page not found.' => 'पृष्ठ नहीं मिला।',
+    'Please fix the following errors:' => 'कृपया निम्नलिखित खामीयां सुधारें:',
+    'Please upload a file.' => 'कृपया एक फ़ाइल अपलोड करें।',
+    'Powered by {yii}' => '{Yii} द्वारा संचालित',
+    'Showing <b>{begin, number}-{end, number}</b> of <b>{totalCount, number}</b> {totalCount, plural, one{item} other{items}}.' => '<b>{totalCount, number}</b> {totalCount, plural, one{चीज} other{चीज़े}} में से दिखा रहा है <b>{begin, number}-{end, number}</b>।',
+    'The combination {values} of {attributes} has already been taken.' => '{attributes} की {values} का संयोजन पहले से ही लिया जा चुका है।',
+    'The file "{file}" is not an image.' => 'फ़ाइल "{file}" एक छवि नहीं है।',
+    'The file "{file}" is too big. Its size cannot exceed {formattedLimit}.' => 'फ़ाइल "{file}" बहुत बड़ी है। इसका माप {formattedLimit} से अधिक नहीं हो सकता है।',
+    'The file "{file}" is too small. Its size cannot be smaller than {formattedLimit}.' => 'फ़ाइल "{file}" बहुत छोटी है। इसका माप {formattedLimit} से छोटा नहीं हो सकता है।',
+    'The format of {attribute} is invalid.' => '{attribute} का प्रारूप अमान्य है।',
+    'The image "{file}" is too large. The height cannot be larger than {limit, number} {limit, plural, one{pixel} other{pixels}}.' => 'छवि "{file}" बहुत बड़ी है। ऊंचाई {limit, number} {limit, plural, one{पिक्सेल} other {पिक्सेलस}} से बड़ी नहीं हो सकती।',
+    'The image "{file}" is too large. The width cannot be larger than {limit, number} {limit, plural, one{pixel} other{pixels}}.' => 'छवि "{file}" बहुत बड़ी है। चौड़ाई {limit, number} {limit, plural, one{पिक्सेल} other{पिक्सेलस}} से बड़ी नहीं हो सकती।',
+    'The image "{file}" is too small. The height cannot be smaller than {limit, number} {limit, plural, one{pixel} other{pixels}}.' => 'छवि "{file}" बहुत छोटी है। ऊंचाई {limit, number} {limit, plural, one{पिक्सेल} other{पिक्सेल}} से छोटी नहीं हो सकती।',
+    'The image "{file}" is too small. The width cannot be smaller than {limit, number} {limit, plural, one{pixel} other{pixels}}.' => 'छवि "{file}" बहुत छोटी है। चौड़ाई {limit, number} {limit, plural, one{पिक्सेल} other{पिक्सेल}} से छोटी नहीं हो सकती।',
+    'The requested view "{name}" was not found.' => 'अनुरोधित दृश्य "{name}" नहीं मिला।',
+    'The verification code is incorrect.' => 'सत्यापन कोड गलत है।',
+    'Total <b>{count, number}</b> {count, plural, one{item} other{items}}.' => 'कुल <b>{count, number}</b> {count, plural, one{चीज} other{चीज़े}}।',
+    'Unable to verify your data submission.' => 'आपके डेटा सबमिशन को सत्यापित करने में असमर्थ।',
+    'Unknown alias: -{name}' => 'अज्ञात उपनाम: - {name}',
+    'Unknown option: --{name}' => 'अज्ञात विकल्प: - {name}',
+    'Update' => 'अद्यतन',
+    'View' => 'अवलोकन',
+    'Yes' => 'हाँ',
+    'Yii Framework' => 'Yii फ़्रेमवर्क',
+    'You are not allowed to perform this action.' => 'आपको यह कार्रवाई करने की अनुमति नहीं है।',
```

Comment:
'You are not allowed to perform this action.' => 'आपको यह करने की अनुमति नहीं है।',